controls/tabview/styling/header-item-styling.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Header Item Styling
+page_title: .NET MAUI TabView Documentation - Header Item Styling
+description: Learn how to style the header items inside the Telerik UI for .NET MAUI TabView.
+slug: tabview-header-item-styling
+tags: tabview, header, item, styling, maui, dotnet maui
+position: 2
+---
+
+# .NET MAUI TabView Header Item Styling
+
+Through `HeaderItemStyle` property of the TabView the following styling properties can be applied to the `TabViewHeaderItem`:
```

Comment:
Always aim to add an introductory sentence to each article. Then, you can continue by introducing the table, list, etc.
```suggestion
The TabView control provides the built-in `HeaderItemStyle` property, which allows you to apply styling properties to the `TabViewHeaderItem`.

The following table summarizes the properties that you can apply to the `TabViewHeaderItem`:
```